### ISSUE TITLE
[BUG] Automatically switches to Test Result Tab on any file save

### ISSUE THREAD

**Issue by DScheglov** [jest-community/vscode-jest#1122]
**Describe the bug**
When I save any file in VS Code, VS Code opens the "Test Results" tab instead of the keep Terminal

**To Reproduce**
1. Edit file
2. Type "Ctrl + S" to Save file
3. Test Results Tab are open

**Expected behavior**
Keep Terminal (or other tab) open

**Screenshots**

**Environment (please complete the following information):**
- vscode-jest version: **v6.2.2**
- `node -v`:  **20.7.0**
- `npm -v` or `yarn --version`: **10.1.0**
- jest or react-scripts (if you haven’t ejected) version:
  - **jest@29.7.0**
  - **ts-jest@29.1.2**
- your vscode-jest settings:
    - jest.jestCommandLine: _is absent in `~/.config/Code/User/settings.json`_
    - jest.runMode: _is absent in `~/.config/Code/User/settings.json`_
    - jest.outputConfig? _is absent in `~/.config/Code/User/settings.json`_
    - jest.autoRevealOutput: "off"
- Operating system: Ubuntu 22.04

**Prerequisite**
- are you able to run jest from the command line?
  - yes
- where do you run jest CLI from?
  - project root
- how do you run your tests from the command line?
  - `npm test`

**Comment by connectdotz** [comment]
Hi, @DScheglov, a few questions: what is the "testing.openTesting" in your settings.json? Did you update it either through the command or manually editing the settings.json?

**Comment by DScheglov** [comment]
Hi, @connectdotz 

`esting.openTesting` is absent in my `settings.json`.
I didn't touch this setting nor manually, nor through the any command

**Comment by connectdotz** [comment]
ok, my guess is the `jest.outputConfig` is saved probably via the command `"Jest: Save Current Output Config"`.  

It's great you figured out a solution with `jest.autoRevealOutput`. However, please know it is a deprecated setting, the right customization is documented in [here](https://github.com/jest-community/vscode-jest/issues/1111#issuecomment-1939656073), and more background info can be found [here](https://github.com/jest-community/vscode-jest?tab=readme-ov-file#outputconfig-issues). 

In short, once you set `"testing.openTesting": "neverOpen"`, you can remove the deprecated `jest.autoRevealOutput` from your settings.json.

I will close this issue as it is a duplicate of #1113 and #1111.
